web/lib/supabase/bets.ts

```diff
@@ -0,0 +1,22 @@
+import { db } from './db'
+import { run } from 'common/supabase/utils'
+import { Bet } from 'common/bet'
+import { JsonData } from 'web/lib/supabase/json-data'
+
+export async function getOlderBets(
+  contractId: string,
+  beforeTime: number,
+  limit: number
+) {
+  const { data } = await run(
+    db
+      .from('bets')
```

Comment:
Note that bets moved to the table `contract_bets` in #1375.

Reply:
Thanks!